helm/values-prod.yaml

```diff
@@ -0,0 +1,246 @@
+global:
+  cloudProvider:
+    dockerRegistryUrl: acrarolibotnonprod.azurecr.io
+    imagePullSecretName: acr-registry
+    flavor: openshift
+  tracing: {}
+  metrics: {}
+  environment: {}
+
+  opalaEnabled: false  
+  authentication:
+    jwtPublicKeySecretName: map-colonies-jwks
+
+enabled: true
+environment: production
+replicaCount: 2
+initialDelaySeconds: 60
+# nodePort: 30030 #for minikube deployment only
+resetOnConfigChange: true
+imagePullPolicy: Always
+
+cloudProvider:
+  dockerRegistryUrl: acrarolibotnonprod.azurecr.io
+  imagePullSecretName: acr-registry
+  flavor: openshift
+
+authentication:
+  ssl:
+    enabled: false
+    useExternal: false
+    externalSecretName: ""
+  opa:
+    enabled: false
+    customHeaderName: "x-api-key,content-type"
+
+caSecretName: ''
+caPath: '/usr/local/share/ca-certificates'
+caKey: 'ca.crt'
+
+image:
+  repository: export-management
+  tag: "v1.5.0"
+
+targetPort: 8080
+internalServicePort: 80
+servicePort: 8080 
+nodePort: 30018 
+
+serviceUrls:
+  exporterTrigger: "http://raster-prod-exporter-trigger-service"
+  jobManager: "http://common-job-manager"
+jobManagement:
+    exporter:
+      jobType: tilesExport
```

Comment:
check how it is defined in raster Prod deployment

Reply:
as written